models/uuvsim_teledyne_whn_urdf/model.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>teledyne_whn</name>
```

Comment:
All of the WHN models have the same name, which makes it a bit confusing.

Reply:
Fixed in latest commit (I think).

Reply:
Looks good.